src/screens/ProfileScreen/ProfileScreen.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import {
+  FlatList,
+  ListRenderItemInfo,
+  SafeAreaView,
+  ScrollView,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import {
+  ExperiencesData,
+  ExperiencesDataItemType,
+  MoreSettingItemType,
+  MoreSettings,
+} from '../../assets';
+import {Button, CustomIcon} from '../../components';
+import {fontStyles} from '../../constants';
+import {useThemeColor} from '../../hooks';
+import {Strings} from '../../i18n';
+import {LocaleUtils, horizontalScale, verticalScale} from '../../utils';
+import ExperienceCard from './components/ExperienceCard/ExperienceCard';
+import ProfileHeader from './components/ProfileHeader';
+
+const ProfileScreen = () => {
+  const {colors} = useThemeColor();
+
+  const experiencesRenderItem = (
+    props: ListRenderItemInfo<ExperiencesDataItemType>,
+  ) => {
+    const {item} = props;
+
+    return (
+      <ExperienceCard
+        logo={item.logo}
+        title={LocaleUtils.localizedText(item.title)}
+      />
+    );
+  };
+
+  const renderExperiences = () => {
+    return (
+      <View style={styles.experiencesSection}>
+        <Text style={fontStyles.b1_Text_Bold}>
+          {Strings.profile.shareYourExperiences}
+        </Text>
+        <FlatList data={ExperiencesData} renderItem={experiencesRenderItem} />
+        <Button
+          onPress={() => {}}
+          mode="outlined"
+          style={[styles.button, {borderColor: colors.buttonBorder}]}>
+          See More
+        </Button>
+      </View>
+    );
+  };
+
+  const horizontalLine = () => {
+    return <View style={styles.horizontalLine} />;
+  };
+
+  const moreSettingsRenderItem = (
+    props: ListRenderItemInfo<MoreSettingItemType>,
+  ) => {
+    const {item} = props;
+
+    return (
+      <TouchableOpacity style={styles.moreSetting}>
+        <CustomIcon name={item.icon} size={verticalScale(25)} />
+        <Text style={[fontStyles.b2_Text_Regular, styles.settingsText]}>
```

Comment:
reverse the order of fontStyles.b2_Text_Regular & styles.settingsText. Because in case someone adds a color style to settingsText by mistake, then the color wont be updated on theme change

Reply:
Fixed.